app/code/Magento/Ui/view/base/web/js/grid/toolbar.js

```diff
@@ -271,12 +271,19 @@ define([
          */
         show: function () {
             this.visible = true;
-            //Check admin grid button has addedr not
-            if ($('.page-main-actions').length === 0) {
+            // Check admin grid button has added or not
+            if ($('.page-main-actions').length === 0 &&
+                typeof this.$sticky !== 'undefined') {
                 this.$sticky.style.top = 0;
             }
-            this.$sticky.style.display = '';
-            this.$toolbar.style.visibility = 'hidden';
+
+            if (typeof this.$sticky !== 'undefined') {
+                this.$sticky.style.display = '';
+            }
+
+            if (typeof this.$toolbar !== 'undefined') {
```

Comment:
Please use underscore library to check if  object is undefined `_.isUndefined(object);` what the purpose to additional check ?

Reply:
app/code/Magento/Braintree/view/frontend/web/js/view/payment/method-renderer/paypal.js 335L

Reply:
please use` isUndefined`